### ISSUE TITLE
Epic: Flaky Spec Cleanup 

### ISSUE THREAD

**Issue by mstruve** [forem/forem#4884]
As a team, we have been noticing recently that we have some flaky specs that we just can't ignore anymore. In an effort to clean them up we are creating this EPIC and plan on linking all new flaky specs that pop up to it.  
### New Issues should include 
* spec file and line number of failing spec
* error message of failure and backtrace
* any other helpful additional information

**Comment by artfullyContrived** [comment]
@mstruve How will we know that this issue is fixed?

**Comment by mstruve** [comment]
Great question! This issue is actually an epic and has 4 related issues already. I suspect this will be an on-going effort for a little bit but you can track its progress by seeing how many issues related to it have been closed.

**Comment by rhymes** [comment]
Found this as well:

```shell
  1) Looking For Work user selects looking for work and autofollows hiring tag

     Failure/Error: expect(user.follows.count).to eq(1)

     

       expected: 1

            got: 0

     

       (compared using ==)

     

     

     

     # ./spec/system/user_selects_looking_for_work_spec.rb:17:in `block (3 levels) in <top (required)>'

     # ./spec/system/user_selects_looking_for_work_spec.rb:15:in `block (2 levels) in <top (required)>'

     # ./spec/rails_helper.rb:90:in `block (3 levels) in <top (required)>'

     # ./spec/rails_helper.rb:90:in `block (2 levels) in <top (required)>'
```

**Comment by mstruve** [comment]
@rhymes can you make a separate issue for that and then mention this issue to link it like the rest? It makes it easier for individuals to pick up and claim when they want to work on something.

**Comment by gayanhewa** [comment]
How do we know which ones are being worked on atm? Before investigating one of the reported issues.

**Comment by mstruve** [comment]
Click on an individual Issue and if no one has said they want to work on it yet then likely it is not being worked on 😊

**Comment by rhymes** [comment]
To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Closing this as we probably don't need an umbrella issue anymore
